source/common/http/filter/ip_tagging_filter.cc

```diff
@@ -9,7 +11,44 @@ IpTaggingFilter::~IpTaggingFilter() {}
 
 void IpTaggingFilter::onDestroy() {}
 
-FilterHeadersStatus IpTaggingFilter::decodeHeaders(HeaderMap&, bool) {
+FilterHeadersStatus IpTaggingFilter::decodeHeaders(HeaderMap& headers, bool) {
+  bool is_internal_request =
+      headers.EnvoyInternalRequest() && (headers.EnvoyInternalRequest()->value() == "true");
```

Comment:
there's a const for the true value in common/http/headers.h

Reply:
fixed.